shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitBusResults.java

```diff
@@ -6,28 +6,35 @@
  */
 package com.powsybl.shortcircuit;
 
-import java.util.Objects;
-
 /**
  * @author Coline Piloquet <coline.piloquet at rte-france.com>
  */
 public class ShortCircuitBusResults {
-    // FIXME
-    // delta voltages are missing for the moment,
-    // but not useful for the global design.
+
+    // VERSION = 1.0 voltageLevelId, busId, voltage
+    // VERSION = 1.1 voltageDropProportional, initialVoltageMagnitude
+    public static final String VERSION = "1.1";
 
     private final String voltageLevelId;
 
     private final String busId;
 
+    private final double initialVoltageMagnitude;
```

Comment:
Any opinion on whether it should be a double or FortescueValue? On my side, I only get the magnitude as a result...